notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    if REGION == "[your-region]":
</pre><p>Should you add a note that if you don't specify a region it'll be in us-central1? Or, do you want to instead throw an error? This line is a little confusing </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I think we should make users choose their region and only have a default for testing, so maybe change this line to <code>if os.getenv("IS_TESTING"):</code></p>

Reply:
<p>In testing environment, the region is defined as <code>_GCP_REGION</code> in <a href="https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/.cloud-build/notebook-execution-test-cloudbuild.yaml#L39" target="_blank">this script</a> , and the implementation for changing region is <a href="https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/4fca7d98b2a239988a832ac674c2de2911d09f4a/.cloud-build/utils/NotebookProcessors.py#L35" target="_blank">here</a> and tested <a href="https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/728dec8526a2df034c5ceade8e456986c543fef6/.cloud-build/utils/UpdateNotebookVariables.py#L75" target="_blank">here</a> . So the region is updated if the content is same as <code>REGION = "[your-region]"</code> , not this condition statement.</p><p> </p><p>Why don't we add a comment like <code># If you do not specify a region, it will be "us-central1".</code> right above the condition statement? </p>